src/audio/mixin_mixout/mixin_mixout.h

```diff
@@ -31,16 +31,18 @@
 #include <sof/platform.h>
 #include <stddef.h>
 
-#define MIXIN_MIXOUT_GENERIC
-
 #if defined(__XCC__)
 
 #include <xtensa/config/core-isa.h>
-#if XCHAL_HAVE_HIFI3 || XCHAL_HAVE_HIFI4
-#undef MIXIN_MIXOUT_GENERIC
-#define MIXIN_MIXOUT_HIFI3
-#endif
-
+# if XCHAL_HAVE_HIFI5
+#  define MIXIN_MIXOUT_HIFI5
+# elif XCHAL_HAVE_HIFI3 || XCHAL_HAVE_HIFI4
+#  define MIXIN_MIXOUT_HIFI3
+# else
+#  define MIXIN_MIXOUT_GENERIC
+# endif
+#else
+# define MIXIN_MIXOUT_GENERIC
```

Comment:
#8787 was merged, this is now available in common.h. Please stop duplicating this.

Reply:
I think we need to fix the kconfig error first.

Reply:
which error?

Reply:
the kconfig error https://github.com/thesofproject/sof/pull/8791